web/react/components/activity_log_modal.jsx

```diff
@@ -49,16 +50,23 @@ export default class ActivityLogModal extends React.Component {
     }
     onShow() {
         AsyncClient.getSessions();
+
+        $(ReactDOM.findDOMNode(this.refs.modalBody)).css('max-height', $(window).height() - 300);
+        if ($(window).width() > 768) {
+            $(ReactDOM.findDOMNode(this.refs.modalBody)).perfectScrollbar();
+        }
```

Comment:
I know the perfectscrollbar stuff is an exception until we have something better but I think the `max-height` stuff should all be handled in `render()` rather than post-render jquery manipulation. We do this in lots of places but we're moving away from doing it since it's very un-React-like